### DISCUSSION TITLE
Inquiry about the Copyright authors of this package

### DISCUSSION THREAD

**Discussion by Sanddrah** [openpgpjs/openpgpjs#1610] [General]
Hello. Please I'd like to know who are the copyright authors of this package. This information is usually on the package.json file but it's not. I will need to add it to the copyright file of the debian package. Thank you. @twiss

**Comment by twiss** [comment]
Hello :wave: It's not entirely trivial to assemble a list of copyright holders to the code of OpenPGP.js. Originally, it was based on [GPG4Browsers](https://web.archive.org/web/20171217133054/http://gpg4browsers.recurity.com/). There is a list of contributors there, and then there are the [contributors to OpenPGP.js itself](https://github.com/openpgpjs/openpgpjs/graphs/contributors). It would be a bit onerous to keep a list of them up to date whenever there are any new contributors, and I imagine you also don't want to list all of them in Debian. Would it be sufficient to just say that the copyright belongs to "the OpenPGP.js contributors"?

**Comment by Sanddrah** [comment]
Yeah I think that will be sufficient. Thanks
